css-basics/css-basics_combined-users/css/styles.css

```diff
@@ -113,3 +113,12 @@ aside .location {
 
   💡 You can use the display property to hide elements.
 */
+
+ul .user .bio {
```

Comment:
I just did "ul .bio" & it worked, but I think your solution is the more specific way

Reply:
Thanks for the hint. Yes, that is true and also works. In that cases it's maybe not necessary to be so specific. The result is the same. :)